nmstatectl/nmstatectl.py

```diff
@@ -394,3 +396,17 @@ def _filter_routes(state, patterns):
                 if fnmatch.fnmatch(route[Route.NEXT_HOP_INTERFACE], pattern):
                     routes[route_type].append(route)
     return routes
+
+
+def _filter_route_rule(state):
+    """
+    return the rules for state's route rule that match the table_id of the
+    filtered route of state by interface
+    """
+    route_rules = {RouteRule.CONFIG: []}
+    route_table_id = state.get(Route.TABLE_ID)
```

Comment:
The state does not contain `Route.TABLE_ID`.

Reply:
Sorry, I'll work on that. Thank you for the example yml file